mozilla_version/gecko.py

```diff
@@ -333,6 +349,71 @@ def _compare(self, other):
     def _compare_version_type(self, other):
         return self.version_type.compare(other.version_type)
 
+    def bump(self, field):
+        """Bump the number defined `field`.
+
+        This function handles previous edge cases.
+
+        Returns:
+            A new GeckoVersion with the right field bumped and the following ones set to 0,
+            if they exist.
+
+        """
+        if field == 'build_number' and self.build_number is None:
+            raise ValueError('Cannot bump the build number if it is not already set')
+        elif field == 'major_number' and self.is_esr:
+            raise ValueError('Cannot predictably know the next major ESR number')
```

Comment:
Ah, this make sense. That being said, I believe we can agree that the bouncer-locations job in the next cut's ESR will likely fail. And that's okay since this is information that is not predictable 👍

Reply:
You comment made me realize we can actually bump the major version if we know it for a fact. I put all the major numbers in a list, and we can expand that list once we know what's the next major ESR. This way, the location job doesn't have to fail, if we update the library in time.